tests/drivers/i2c/i2c_target_api/boards/stm32h573i_dk.overlay

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* I2C bus pins are exposed on the ST morpho header.
+ *
+ *  Bus        SDA               SCL
+ *          Pin   Hdr         Pin   Hdr
+ *  i2c1    PB7   CN13:9      PB6   CN13:10
+ *  i2c2    PB11  CN15:1      PB10  CN15:2
+ *
+ * Short Pin PB7 to PB11, and PB6 to PB10, for the test to pass.
+ */
+
+&i2c1 {
+       clocks = <&rcc STM32_CLOCK_BUS_APB1 0x00200000>,
+		<&rcc STM32_SRC_PLL3_R I2C1_SEL(1)>;
+	pinctrl-0 = <&i2c1_scl_pb6 &i2c1_sda_pb7>;
+	pinctrl-names = "default";
+	clock-frequency = <I2C_BITRATE_FAST>;
+	status = "okay";
```

Comment:
Not required as provided by board, unless there is a specific reason to duplicate it

Reply:
Yes you are right there no specific reason to duplicate, i will remove it